yapapi/contrib/service/http_proxy.py

```diff
@@ -262,14 +258,15 @@ async def _request_handler(self, request: web.Request) -> web.Response:
         return await instance.handle_request(request)
 
     async def run(self):
+        """Run a local HTTP server.
```

Comment:
well, it's not just _any_ HTTP server...

Reply:
`Run a local HTTP proxy server.`